StoreEycaExpirationActivity/table.ts

```diff
@@ -0,0 +1,28 @@
+import { TableService, TableUtilities } from "azure-storage";
+import * as date_fns from "date-fns";
+import { TaskEither } from "fp-ts/lib/TaskEither";
+import { taskify } from "fp-ts/lib/TaskEither";
+import { FiscalCode, NonEmptyString } from "italia-ts-commons/lib/strings";
+
+const eg = TableUtilities.entityGenerator;
+
+export const insertEycaExpiration = (
+  tableService: TableService,
+  cgnExpirationTableName: NonEmptyString
+) => (
+  fiscalCode: FiscalCode,
+  activationDate: Date,
+  expirationDate: Date
+): TaskEither<Error, TableService.EntityMetadata> =>
```

Comment:
Maybe we can refactor this method with a general `insertCardExpiration` and let it to be used both by EycaCard and CGN. This means that we can move table utility at the top level (into `utils` folder).

Reply:
Yes absolutely, I totally agree

Reply:
Done in [#f726380](https://github.com/pagopa/io-functions-cgn/pull/27/commits/f7263803f453dcfc5859adcf2e30efcfa48b57ab)